modules/proc_installing-the-rollout-manager-operator.adoc

```diff
@@ -0,0 +1,42 @@
+:_mod-docs-content-type: PROCEDURE
+[id="installing-the-rollout-manager-operator_{context}"]
+= Installing the rollout-manager Operator
+
+You can install the `rollout-manager` Operator by defining the namespace and using the `Deploy` Operator.
+
+.Prerequisites
+
+* The `cluster-admin` cluster role must be assigned to the user.
+
+.Procedure
+
+. Defining the namespace - By default, the `rollout-manager` Operator is installed in the `argo-rollouts-manager-system` namespace. To modify the `rollout-manager` Operator, update the value of the `namespace` specified in the `config/default/kustomization.yaml` file.
+
+. Using the `Deploy` Operator - Deploy the `rollout-manager`:
++
+[source,terminal]
+----
+make deploy
+----
+
+. After the `rollout-manager` Operator is installed, you can use this Operator with the following configuration.
```

Comment:
This page can/should be removed. [This page](https://docs.openshift.com/gitops/1.12/argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.html#gitops-about-argo-rollout-manager-custom-resources-and-spec_using-argo-rollouts-for-progressive-deployment-delivery
) contains newer, more correct information.

Reply:
Fixed that change now @jgwest. PTAL.